src/plugins/notebook/components/SectionCollection.vue

```diff
@@ -75,21 +75,25 @@ export default {
 
             const selectedSection = this.sections.find(s => s.id === this.selectedSectionId);
             const defaultNotebook = getDefaultNotebook();
-            const defaultSection = defaultNotebook && defaultNotebook.section;
+            const defaultSectionId = defaultNotebook && defaultNotebook.defaultSectionId;
             const isSectionSelected = selectedSection && selectedSection.id === id;
```

Comment:
Since we deleted the section on line 74 above, will `selectedSection && selectedSection.id === id` ever match?
Perhaps we need to do this check before deleting the notebook section?

Reply:
it actually is async action so ll take some time before it actually deletes it.